File: programmers/가운데글자가져오기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s)
{
    string answer = "";
    if (s.size() % 2 == 1)
    {
        int mid = s.size() / 2;
        answer.push_back(s[mid]);
    }
    else
    {
        int mid = s.size() / 2 - 1;
        answer.push_back(s[mid]);
        answer.push_back(s[mid + 1]);
    }
    return answer;
}

File: programmers/가운데글자가져오기.js
// https://programmers.co.kr/learn/courses/30/lessons/12903?language=javascript
function solution(s) {
  var answer = "";
  if (s.length % 2 == 1) {
    return s[parseInt(s.length / 2)];
  } else {
    return s[parseInt(s.length / 2) - 1] + s[parseInt(s.length / 2)];
  }
}
